### ISSUE TITLE
Environment variable configured by CQtDeployer's own installer on Windows should not be quoted

### ISSUE THREAD

**Issue by pzhlkj6612** [QuasarApp/CQtDeployer#581]
**Describe the bug**
Because of the quotes, the environment variable configured by CQtDeployer's own installer doesn't work properly in PowerShell on Windows.

**To Reproduce**
1. Open PowerShell.
2. Check the environment variables configured by CQtDeployer's installer via `Get-Item -Path Env:\cqt*` ([Getting environment variables](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_provider#getting-environment-variables)).
3. Try to call CQtDeployer via `& ${Env:\cqtdeployer}` , failed ([Call operator `&`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators#call-operator-) ).
4. Try to change the working directory to the installation location via `Set-Location -Path ${Env:\cqtDir}` , failed ([Set-Location](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/set-location)).
5. If I remove these quotes, everything works fine.

**Expected behavior**
Use those variables normally.

**Console output**

Errors:

```console
PS C:\> Get-Item -Path Env:\cqt*

Name                           Value
----                           -----
cqtDir                         "C:\Program Files (x86)\CQtDeployer\1.4\"
CQTDEPLOYER_VERSION            1.4.7
cqtdeployer                    "C:\Program Files (x86)\CQtDeployer\1.4\cqtdeployer.bat"

PS C:\> & ${Env:\cqtdeployer}
&: The term '"C:\Program Files (x86)\CQtDeployer\1.4\cqtdeployer.bat"' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

PS C:\> Set-Location -Path ${Env:\cqtDir}
Set-Location: Cannot find drive. A drive with the name '"C' does not exist.
```

After removing quotes:

```console
PS C:\> ${Env:\cqtdeployer} = ${Env:\cqtdeployer}.Trim('"')
PS C:\> ${Env:\cqtDir} = ${Env:\cqtDir}.Trim('"')

PS C:\> Get-Item -Path Env:\cqt*

Name                           Value
----                           -----
cqtDir                         C:\Program Files (x86)\CQtDeployer\1.4\
CQTDEPLOYER_VERSION            1.4.7
cqtdeployer                    C:\Program Files (x86)\CQtDeployer\1.4\cqtdeployer.bat

PS C:\> & ${Env:\cqtdeployer}
Info: Print info ...
...

PS C:\> Set-Location -Path ${Env:\cqtDir}
PS C:\Program Files (x86)\CQtDeployer\1.4>
```

**Desktop:**
 - OS: Windows 10 2004 10.0.19041.867
 - PowerShell
   - 5.1.19041.610
   - 7.1.2

**Additional context**
Some code in project may need to be modified:

- https://github.com/QuasarApp/CQtDeployer/blob/78845ef8d5ad3f7af7cfdedbcdabeb7cd00ae5c6/test.pri#L11
- https://github.com/QuasarApp/CQtDeployer/blob/78845ef8d5ad3f7af7cfdedbcdabeb7cd00ae5c6/QIFData/InstallerBase.pri#L7

Diff:

```diff
-  win32:DEPLOYER=$$(cqtdeployer)
+  win32:DEPLOYER="$$(cqtdeployer)" 
```

Let me know if you have any questions about ~PowerShell~ this issue (sorry).

**Comment by EndrII** [comment]
oh, but this will not break the launch from cmd?

**Comment by pzhlkj6612** [comment]
> Diff:
> 
> ```diff
> -  win32:DEPLOYER=$$(cqtdeployer)
> +  win32:DEPLOYER="$$(cqtdeployer)" 
> ```

Wait, the above code will break the execution since the quotes will be added unconditionally, regardless of whether the value of `cqtdeployer` is already quoted. We need a more intelligent function.

**Comment by pzhlkj6612** [comment]
> oh, but this will not break the launch from cmd?

Yes, it works fine in cmd.

**Comment by EndrII** [comment]
merged into 1.5
